vignettes/posterior.Rmd

```diff
@@ -12,41 +12,65 @@ vignette: >
   %\VignetteEncoding{UTF-8}
 ---
 
-```{r child="children/settings-knitr.Rmd"}
+```{r child="children/_settings-knitr.Rmd"}
+```
+
+<!-- 
+NOTE: the hack below of using print.data.frame in chunks with echo=FALSE 
+is used because the pillar formatting of posterior draws_summary objects
+isn't playing nicely with pkgdown::build_articles(). When that is fixed
+using options(digits=2) also won't be necessary anymore.
```

Comment:
I had to use an ugly hack to avoid a formatting issue where draws_summary objects from posterior aren't playing nicely with `pkgdown::build_article`. At some point we should revisit this and figure out what's going on.